src/current/cockroachcloud/frequently-asked-questions.md

```diff
@@ -81,7 +81,7 @@ The Cockroach Labs SRE team has direct access to {{ site.data.products.db }} clu
 
 ### How do I change the configurations on my cluster?
 
-Contact [Support](https://support.cockroachlabs.com/hc/en-us) to change your cluster configuration.
+You can [change your cluster's compute](cluster-management.html#change-compute-for-a-cluster), [add and remove nodes](cluster-management.html#add-or-remove-nodes-from-a-cluster), and [increase storage](cluster-management.html#increase-storage-for-a-cluster) from the {{ site.data.products.db }} Console. Due to cloud provider limitations, storage space cannot be removed from a node once added.
```

Comment:
I think you can also make these changes using the API or Terraform, right?

Reply:
double checking this with mark, but good point